README.md

```diff
@@ -84,6 +84,21 @@ two properties in the "Common" section
 copy of the `./config/marklogic-connect-standalone.properties` file. This file already has the two properties set to the
   correct values, so you should not need to do anything further with them.
 
+### Dead Letter Configuration
+
+Starting with version 1.8.0, the sink connector makes use of the dead letter queue (DLQ) if the user has configured
+Kafka to use the DLQ. For information about configuring Kafka to use the DLQ, please refer to:
+
+    https://www.confluent.io/blog/kafka-connect-deep-dive-error-handling-dead-letter-queues/
+
+When Kafka has been configured to use the DLQ, there are two events in the sink connector that will cause a record to be
+sent to the DLQ.
+- A conversion problem: If a specific record cannot be converted into the target format to be delivered to MarkLogic,
+then that record will be sent to the DLQ.
+- A write problem: If a batch of documents (converted Kafka records from the source topic) fails to be written to
+MarkLogic then each of the records in the batch will be sent to the DLQ. The entire batch must be sent to the DLQ since
+the connector is unable to determine the cause of the failure.
```

Comment:
Is it not worth documenting the headers that are added? Those seem worth documenting to me, as it's our attempt to provide some assistance with trying to figure out why the record failed.

Reply:
Sorry, forgot this one.